docs/guide/personalization.md

```diff
@@ -186,43 +178,50 @@ ez_recommendation:
         script_url: cdn.yoochoose.net/yct.js
 ```
 
-Note: changing any of these parameters without a valid reason will break all calls to the Personalization Engine.
+!!! caution
+
+    Changing any of these parameters without a valid reason will break all calls to the Recommendation engine.
 
 #### Enable tracking
 
-The `EzSystemsRecommendationBundle` delivers a Twig extension which helps integrate the tracking functionality into your site. Place the following code snippet in the HEAD section of your header template:
+The `EzRecommendationClientBundle` delivers a Twig extension
+which helps integrate the tracking functionality into your site.
```

Comment:
There is only user tracking provided by Twig extension so maybe it should be "which helps integrate the user tracking functionality into your site."